helm/github-exporter-chart/templates/psp.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
```

Comment:
Do we need a PSP? What calls does the application make to the Kubernetes API?

Reply:
I think we need this for the configmap and secret. We declare volumes below. Not sure if it works without. Only copied from some operator.

Reply:
We'd need this if the Pod was requesting the configmap and/or the secret via the API. We don't need it in this case.

Reply:
It says this. Doesn't look like API request. I understand this as being necessary in order to mount volumes into the pod. But I am not sure though. 

```
  volumes:
    - 'secret'
    - 'configMap'
```